web/js/notifications/wv.notifications.ui.js

```diff
@@ -151,7 +157,7 @@ wv.notifications.ui = wv.notifications.ui || function(models, config) {
    *  in LocalStorage
    */
   var getNumberOfTypeNotseen = function(type, arra) {
-    var storageItem = wv.util.browser.localStorage?!!localStorage.getItem(type):false;
+    var storageItem = localStorage.getItem(type);
```

Comment:
You need to check `wv.util.browser.localStorage` here before you use `localStorage`. This was the problem that was causing a crash before.